ingest/src/test/java/com/microsoft/azure/kusto/ingest/E2ETest.java

```diff
@@ -73,21 +78,20 @@ public static void setUp() {
     @AfterAll
     public static void tearDown() {
         try {
-            queryClient.execute(databaseName, String.format(".drop table %s ifexists", tableName));
+            queryClient.executeForJsonResult(databaseName, String.format(".drop table %s ifexists", tableName));
```

Comment:
For performance, the new default should be to use "executeForJsonResult" for any query where we don't care to see the result.